tests/testsuite/kubevirtresource.go

```diff
@@ -51,6 +56,41 @@ var (
 	originalKV            *v1.KubeVirt
 )
 
+/*
+TranslateBuildArch translates the build_arch to arch
+
+	case1:
+	  build_arch is crossbuild-s390x, which will be translated to s390x arch
+	case2:
+	  build_arch is s390x, which will be translated to s390x arch
+*/
+func TranslateBuildArch() string {
+	buildArch := os.Getenv("BUILD_ARCH")
+
+	if buildArch == "" {
+		return ""
+	}
+	archElements := strings.Split(buildArch, "-")
+	if len(archElements) == 2 {
+		return archElements[1]
+	}
+	return archElements[0]
+
+}
+
+// GetVMGuestByArchitecture returns the VMGuest based on the architecture.
+func GetVMGuestByArchitecture(arch string,
+	vmi *v1.VirtualMachineInstance, loginToVMI console.LoginToFunction, opts ...libvmi.Option) (*v1.VirtualMachineInstance, console.LoginToFunction) {
```

Comment:
I think this function should either always be used to construct the VMI, thus enabling better arch specific options, or alternatively be renamed into something more specific like `ReplaceVMGuestOSIfS390X`

Reply:
done